### DISCUSSION TITLE
Question about *_nix.c.v and *_windows.c.v files

### DISCUSSION THREAD

**Discussion by mike-ward** [vlang/v#21659] [Questions and Answers] [answered]
I'm not seeing build rules for these `*_platform` files. Does the V app detect the platform and only use the matching platform file? If so, are there platforms other than `windows` and `nix`?

**Comment by JalonSolov** [comment, marked as answer]
V will use the current platform, unless you tell it to compile for something else, using `-os` and/or `-target` options.

For platforms it knows about, see https://github.com/vlang/v/blob/master/vlib/v/pref/should_compile.v
